holoviews/core/dimension.py

```diff
@@ -1388,7 +1388,7 @@ def options(self, *args, **kwargs):
         Returns:
             Returns the cloned object with the options applied
         """
-        backend = kwargs.pop('backend', None)
+        backend = kwargs.get('backend', None)
```

Comment:
Now that the backend is still in kwargs won't the ``elif args and kwargs:`` branch execute and error?

Reply:
I don't think there is any issue here - the point of the PR is that `backend` is now like any other kwarg. It is just that the value is needed here (if available) for `obj._dispatch_opts`.

Reply:
It will now error when you supply args and the backend as a kwarg as far as I can tell.

Reply:
e.g. as far as I can tell ``.options(opts.Image(cmap='Reds'), backend='matplotlib')`` will get caught up in the ``elif args and kwargs`` branch and raise a ValueError. Maybe I'm missing something?

Reply:
I don't think `.options` is supposed to accept `Option` objects afaik. Only `.opts` should.

Reply:
I didn't think that was true, I thought they were basically equivalent except for clone and that ``.options`` doesn't allow the really old nested format. In any case though the point I'm making, the same problem applies to ``.opts({'Image': dict(cmap='Reds')}, backend='matplotlib')``

Reply:
Addressed in f112875